decidim-proposals/app/views/decidim/participatory_processes/participatory_process_groups/_proposal.html.erb

```diff
@@ -1,27 +1 @@
-<div class="column">
-  <article class="card card--proposal">
-    <div class="card__content">
-      <div class="card__header">
-        <%= link_to resource_locator(proposal).path do%>
-            <h5 class="card__title"><%= proposal.title %></h5>
-        <% end %>
-        <div class="card__author author-data author-data--small">
-          <%= render partial: "decidim/shared/author_reference", locals: { author: present(proposal).author } %>
-        </div>
-        <div class="tech-info tech-info--text-left">
-          <%= t(".creation_date", date: l(proposal.created_at, format: :decidim_short)) %>
-        </div>
-        <%= resource_reference(proposal, class: "tech-info--text-left") %>
-      </div>
-      <%= render partial: "decidim/proposals/proposals/proposal_badge", locals: { proposal: proposal } %>
-      <p><%= truncate(proposal.body, length: 100) %></p>
-      <%= render partial: "decidim/shared/tags", locals: { resource: proposal, tags_class_extra: "tags--proposal" } %>
-    </div>
-    <div class="card__footer">
-      <div class="card__support">
-        <div class="card__support__data"></div>
-        <%= link_to t(".view_proposal"), resource_locator(proposal).path, class: "card__button button small secondary" %>
-      </div>
-    </div>
-  </article>
-</div>
+<%= cell "decidim/proposals/proposal_m", proposal %>
```

Comment:
Maybe we can remove this whole file and make the parent view render the cell directly instead of rendering this partial?